frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java

```diff
@@ -1,8 +1,7 @@
 package com.google.tripmeout.frontend;
 
-import javax.annotation.Nullable;
-
 import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
```

Comment:
Make sure to run the formatter. (Not sure if this is the result of the formatter or incorrect import reordering.)

Reply:
I think it was the formatter but I've switched them

Reply:
Assume that whatever the formatter does is correct. Just wanted to make sure you'd run it.